step_0:
  tf_mat_link2_0_flan2:
    target_pose:
      position: [0.4239172719191898, -0.02367375359383433, 0.3479214373421741]
      orientation:
        - [0.9849253322778994, -0.113675802400225, 0.1303836983126263]
        - [-0.1716578852638999, -0.7353229410282894, 0.6556170298935043]
        - [0.02134633256958956, -0.6681152111842278, -0.7437515396084561]
  tf_mat_link3_0_flan3:
    target_pose:
      position: [0.4845416764727903, 0.1040752691736836, -0.1791903810381915]
      orientation:
        - [-0.9849269989698554, -0.1136755555309222, -0.1303713224656858]
        - [-0.0213372508262374, -0.6681147936306463, 0.7437521752677776]
        - [-0.1716494509872828, 0.7353233585827778, 0.6556187698267569]
step_1:
  tf_mat_link2_0_flan2:
    target_pose:
      position: [0.4269912644725464, -0.005394885913313885, 0.364573299165429]
      orientation:
        - [0.9880361523353487, -0.08236927840341352, 0.1303835480047918]
        - [-0.1482412329716264, -0.7404008628564491, 0.6556150136514334]
        - [0.04253355586796248, -0.667099553618592, -0.7437533428933012]
  tf_mat_link3_0_flan3:
    target_pose:
      position: [0.4876155825302161, 0.120726208801136, -0.1974682424640844]
      orientation:
        - [-0.9880378108368412, -0.08236900915156675, -0.1303711493053341]
        - [-0.0425244626835705, -0.6670992510603628, 0.7437541342020155]
        - [-0.1482327872844066, 0.7404011654156673, 0.6556165815557217]
step_2:
  tf_mat_link2_0_flan2:
    target_pose:
      position: [0.4300652355241789, 0.0128839843480657, 0.3812251595141942]
      orientation:
        - [0.9901521150288115, -0.05097981600733711, 0.1303834828688615]
        - [-0.1246752580309982, -0.7447332661625901, 0.6556129874058791]
        - [0.06367788759085834, -0.6654121806599835, -0.7437551401513544]
  tf_mat_link3_0_flan3:
    target_pose:
      position: [0.4906895100893807, 0.137377149902999, -0.2157461013091973]
      orientation:
        - [-0.9901537646344906, -0.05097952464404161, -0.1303710686255808]
        - [-0.06366878661979043, -0.6654119934017554, 0.7437560867947025]
        - [-0.1246668045875718, 0.7447334534218883, 0.655614382184557]
step_3:
  tf_mat_link2_0_flan2:
    target_pose:
      position: [0.4331391404917645, 0.03116287095670622, 0.3978770227036758]
      orientation:
        - [0.9912710897633924, -0.01953902173725131, 0.130383502970422]
        - [-0.100983689379221, -0.7483157885841853, 0.6556109531970984]
        - [0.08475803719031624, -0.6630547913497989, -0.7437569295729299]
  tf_mat_link3_0_flan3:
    target_pose:
      position: [0.4937634146057028, 0.1540280977088764, -0.234023972369548]
      orientation:
        - [-0.9912727297768662, -0.01953870855586655, -0.1303710805076637]
        - [-0.08474893209477682, -0.6630547195801234, 0.7437580310797431]
        - [-0.1009752318417941, 0.7483158603550112, 0.6556121739278433]
step_4:
  tf_mat_link2_0_flan2:
    target_pose:
      position: [0.4362130239415193, 0.04944175911434573, 0.4145288835063782]
      orientation:
        - [0.9913919498264793, 0.01192144619474675, 0.1303836082892326]
        - [-0.07719038241571191, -0.751144822825131, 0.6556089130733678]
        - [0.1057527787443278, -0.6600297593790508, -0.7437587093562318]
  tf_mat_link3_0_flan3:
    target_pose:
      position: [0.4968373406398701, 0.1706790479014538, -0.2523018418809787]
      orientation:
        - [-0.991393579561014, 0.01192178087889229, -0.1303711849396186]
        - [-0.1057436731906743, -0.660029803170192, 0.7437599650994078]
        - [-0.07718192445061293, 0.7511447790352191, 0.6556099590091082]
step_5:
  tf_mat_link2_0_flan2:
    target_pose:
      position: [0.4392869304613259, 0.06772063402116291, 0.4311807366972098]
      orientation:
        - [0.9905145735222424, 0.04336990976685165, 0.1303837987192462]
        - [-0.05331929498173477, -0.7532175202879882, 0.6556068690889176]
        - [0.1266409723295396, -0.6563401306982614, -0.7437604777091693]
  tf_mat_link3_0_flan3:
    target_pose:
      position: [0.4999113327309038, 0.1873299961610134, -0.2705796960808909]
      orientation:
        - [-0.9905161923014547, 0.04337026561677809, -0.1303713818162912]
        - [-0.1266318699845906, -0.6563402900061235, 0.7437618869063027]
        - [-0.0533108402557228, 0.7532173609814331, 0.655607739658586]
step_6:
  tf_mat_link2_0_flan2:
    target_pose:
      position: [0.4423607922444357, 0.08599951058724342, 0.4478325835258433]
      orientation:
        - [0.9886398450720598, 0.07477469275995208, 0.1303840740686119]
        - [-0.02939447108623694, -0.7545317936359074, 0.6556048233025387]
        - [0.1474015785247033, -0.6519896219845239, -0.7437622328505877]
  tf_mat_link3_0_flan3:
    target_pose:
      position: [0.5029853229797731, 0.2039809446375869, -0.2888575509185945]
      orientation:
        - [-0.9886414522306013, 0.07477506941736062, -0.1303716709393337]
        - [-0.1473924830520439, -0.6519898966486581, 0.743763794564708]
        - [-0.02938602326280668, 0.7545315189731567, 0.6556055181117035]
step_7:
  tf_mat_link2_0_flan2:
    target_pose:
      position: [0.4454346346633129, 0.1042783991287132, 0.4644844375362253]
      orientation:
        - [0.9857696498355426, 0.106104204065929, 0.1303844340603885]
        - [-0.005439977456731054, -0.7550863204259496, 0.6556027777721515]
        - [0.1680137135020525, -0.6469826092302958, -0.7437639730149281]
  tf_mat_link3_0_flan3:
    target_pose:
      position: [0.5060593366237354, 0.2206319077673539, -0.3071354177494554]
      orientation:
        - [-0.9857712447197539, 0.1061046011515897, -0.130372052017953]
        - [-0.1680046285583565, -0.6469829989742122, 0.7437656861556425]
        - [-0.005431540192435469, 0.7550859306834917, 0.6556032966031846]
step_8:
  tf_mat_link2_0_flan2:
    target_pose:
      position: [0.4485084537764359, 0.1225572751563619, 0.4811362764499152]
      orientation:
        - [0.98190688161724, 0.137326856466875, 0.1303848783316231]
        - [0.01852003437110289, -0.7548805415699628, 0.6556007345601134]
        - [0.1884565955777396, -0.6413241406344264, -0.7437656964477104]
  tf_mat_link3_0_flan3:
    target_pose:
      position: [0.5091333695426182, 0.2372828641587142, -0.3254132731257476]
      orientation:
        - [-0.9819084635858371, 0.1373272735809617, -0.1303725246679362]
        - [-0.1884475248090648, -0.641324645065609, 0.7437675597719542]
        - [0.01852845743035686, 0.7548800371403124, 0.6556010773728124]
step_9:
  tf_mat_link2_0_flan2:
    target_pose:
      position: [0.4515822537251625, 0.1408361621197867, 0.4977881216638847]
      orientation:
        - [0.9770554248347125, 0.168411252319344, 0.1303854064355544]
        - [0.04246147009871917, -0.7539146640001155, 0.6555986957210876]
        - [0.2087096672951402, -0.6350199063790684, -0.7437674014158441]
  tf_mat_link3_0_flan3:
    target_pose:
      position: [0.5122074256567578, 0.2539338360847379, -0.3436911415348094]
      orientation:
        - [-0.9770569932593994, 0.1684116890418896, -0.130373088413984]
        - [-0.2087006143332904, -0.6350205249896709, 0.7437694135295273]
        - [0.04246987532130919, 0.7539140453911171, 0.6555988626522546]